apidoc/Snapshotter.yml

```diff
@@ -0,0 +1,132 @@
+    summary: When this property is set to YES, the map displays icons and labels for 
+        restaurants, schools, and other relevant points of interest.
+    type: Boolean
+    default: true
+
+  - name: mapType
+    summary: |
+        Map type constant, either <Modules.Map.NORMAL_TYPE>, <Modules.Map.SATELLITE_TYPE>,
+        <Modules.Map.TERRAIN_TYPE>, <Modules.Map.HYBRID_TYPE>, <Modules.Map.HYBRID_FLYOVER_TYPE> or 
+        <Modules.Map.SATELLITE_FLYOVER_TYPE>.
+    type: Number
+    default: NORMAL_TYPE
+
+  - name: showsBuildings
+    summary: Determines whether building will be shown on the map. The mapType property must be set 
+        to <Modules.Map.STANDARD_TYPE> for extruded buildings to be displayed.
+    type: Boolean
+    default: true
+
+ examples:
+  - title: Snapshot without a mapView
+    example: |
+       
+                var MapModule = require('ti.map');
+
+                var win = Ti.UI.createWindow({
+                    backgroundColor: 'white'
+                });
+
+
+                var Snapshotter = MapModule.createSnapshotter({
+                    mapType: MapModule.HYBRID_TYPE,
+                    region: {
+                        latitude: 37.3382,
+                        longitude: -121.8863,
+                        latitudeDelta: 0.4,
+                        longitudeDelta: 0.4
+                    }
+                });
+
+
+                Snapshotter.takeSnap(function(e) {
+                    var imageView = Ti.UI.createImageView({
+                        image: e.image,
+                        width:Ti.UI.FIlL,
+                        height:Ti.UI.FILL
+                    });
+                    win.add(imageView);
+                });
+
+                win.open();
+
+
+  - title: Snapshot with mapView
+    example: |
+                var MapModule = require('ti.map');
+                var win = Ti.UI.createWindow({
+                    backgroundColor: 'white'
+                });
+
+
+                var mapview = MapModule.createView({
+                    mapType: MapModule.HYBRID_TYPE,
+                    region: {
+                        latitude: 37.389569,
+                        longitude: -122.050212,
+                        latitudeDelta: 0.2,
+                        longitudeDelta: 0.2
+                    }
+                });
+                win.add(mapview);
+
+                var Snapshotter = MapModule.createSnapshotter({})
+
+                var imageView = Ti.UI.createImageView({
+                        width:Ti.UI.FIlL,
+                        height:Ti.UI.FILL
+                });
+
+                var button = Titanium.UI.createButton({
+                   title: 'Take Snap!',
+                   top: 10,
+                   width: 100,
+                   height: 50
+                });
+
+                button.addEventListener('click',function(e) {
+                   Snapshotter.takeSnap(function(e) {imageView.image = e.image},mapview);
```

Comment:
Please format correctly. Also consider to refactor the method signature to something more readable, e.g. object:
```js
Snapshotter.takeSnapshot({
    success: function(e) {
        imageView.image = e.image;
    },
    error: function() {
    
    },
    mapView: mapview
});
```